### PR TITLE
Count 3.1.4xx and 5.0.0-preview 4 in

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1047,11 +1047,11 @@
         }
       }
     },
-    // Automate opening PRs to merge cli 3.1.3xx to release/5.0.1xx-preview3
+    // Automate opening PRs to merge cli 3.1.3xx to 3.1.4xx
     {
       "triggerPaths": [
-        "https://github.com/dotnet/sdk/blob/release/3.1.3xx/**/*",
-        "https://github.com/dotnet/installer/blob/release/3.1.3xx/**/*"
```
Comment (sfoslund) [inline Maestro/subscriptions.json:1054]: Shouldn't this stay as 3.1.3xx? So we have 3.1.3xx-> 3.1.4xx?

Comment (wli3) [reply Maestro/subscriptions.json:1054]: ah, ya we should disable that. 3.1.3xx will get the latest runtime soon

**Comment by wli3** [comment]
@wtgodbe could you help me merge this?

**Comment by wli3** [comment]
@mthalman could you help me merge this?

**Pull request merged**